spec/lib/metadata/client_spec.rb

```diff
@@ -92,43 +92,121 @@
     it { should be_a Metaforce::Job::Retrieve }
   end
 
-  describe '._create' do
+  describe '._create_metadata' do
     before do
-      savon.expects(:create).with(:metadata => [{:full_name => 'component', :label => 'test', :content => "Zm9vYmFy\n"}], :attributes! => {'ins0:metadata' => {'xsi:type' => 'ins0:ApexComponent'}}).returns(:in_progress)
+      savon.expects(:create_metadata).
+          with({
+                 :metadata =>
+                 [
+                   {
+                     :full_name => 'component',
+                     :label => 'test',
+                     :content => "Zm9vYmFy\n"
+                   }
+                 ],
+                 :attributes! =>
+                 {
+                   'ins0:metadata' => { 'xsi:type' => 'ins0:ApexComponent' }
+                 }
+               }).returns(:result)
```

Comment:
Yes.  Savon has some built-in fixture handling (see the renaming of various XML files which are inferred from this symbol), and I wanted to clarify that the call is now synchronous and returns an actual result instead of information about an in-progress job.